packages/eas-cli/src/build/build.ts

```diff
@@ -177,6 +177,15 @@ async function uploadProjectAsync<TPlatform extends Platform>(
           )}`
         );
         const projectTarball = await makeProjectTarballAsync();
+
+        if (projectTarball.size > 1024 * 1024 * 100) {
+          Log.warn(
+            `Your project size is over 100 MB. You can reduce its size by excluding files unnecessary for the build process using ${chalk.bold(
```

Comment:
```suggestion
            `Your project archive is ${projectTarball.size}. You can reduce its size and the time it takes to upload by excluding files that are unnecessary the build process in ${chalk.bold(
```

let's also pretty format the `projectTarball.size` here so that it reads more nicely :)

the changes here have two purposes:
- show the full size rather than at what point we show this warning, because showing 100 MB here seems to indicate that at 100 MB there is some kind of cut off. we just want to convey that the archive is large
- explain that the only downside of a large project is that it will take longer to upload

Reply:
It makes sense, thank you for your suggestion!